README.md

```diff
@@ -23,7 +23,7 @@ Shopify’s TSLint rules come bundled in `tslint-config-shopify`.
 To enable these rules, create a `tslint.json` file at the root level of your project, and extend `tslint-config-shopify`.
 ```
 {
-  "extends": "tslint-config-shopify"
+  "extends": "tslint-config-shopify/base"
```

Comment:
The base rules will include all typed rules, so just 3 configs.  With the assumption that I'm predicting `extends`' behaviour correctly.